modules/tinymce/src/themes/silver/main/ts/ReadOnly.ts

```diff
@@ -46,21 +34,9 @@ const broadcastReadonly = (uiComponents: RenderUiComponents, readonly: boolean)
   uiComponents.uiMothership.broadcastOn([ ReadOnlyChannel ], { readonly });
 };
 
-export const toggleToReadOnly = (uiComponents: RenderUiComponents, readonly: boolean) => {
-  const outerContainer = uiComponents.outerContainer;
-
-  broadcastReadonly(uiComponents, readonly);
+const toggleToReadOnly = (uiComponents: RenderUiComponents, readonly: boolean) => broadcastReadonly(uiComponents, readonly);
```

Comment:
This function really isn't needed anymore, we can just call `broadcastReadonly` instead.